modules/eventing/pages/eventing-language-constructs.adoc

```diff
@@ -375,52 +375,40 @@ The `close()` method on the iterable handle can throw an exception if the underl
 |===
 
 [#analytics_call]
-=== `ANALYTICS()`
+=== `couchbase.{zwsp}analyticsQuery({wj})`
 
 ifeval::['{page-component-version}' == '7.6'] 
 _(Introduced in Couchbase Server 7.6)_ 
 endif::[]
 
-The `ANALYTICS()` function provides integration with {sqlpp} Analytics directly from the Eventing Service.
+The `couchbase.analyticsQuery()` function provides integration with {sqlpp} Analytics directly from the Eventing Service.
 
 Integrating Eventing with Analytics:
 
 * Allows Eventing to benefit from the high availability and load balancing of Analytics, where requests can take turns being submitted across nodes
 * Simplifies Eventing code logic and improves code readability
 * Eliminates security and network latency issues with the `curl()` function
 
+The following example assumes that the Analytics collection (dataset) called `default` already exists.
+
 [source,javascript]
 ----
 function OnUpdate(doc, meta) {
-    // Ignore information we don't care about
-    if (doc.type !== 'airline') return;
-
-    // Get the total routes per IATA
-    var route_cnt = 0;     
-    // Uses a true variable as a SQL++ parameter  
-    var airline = doc.iata;  
-
-    var results = ANALYTICS(
-        "SELECT COUNT(*) AS cnt
-        FROM `travel-sample`.`inventory`.`route`
-        WHERE type = \"route\" 
-		AND airline = $1", [doc.iata]
-    );
+    var count = 0;
+    const limit = 4;
 
-    // Stream results using the 'for' iterator
-    for (var item of results) {   
-        route_cnt = item.cnt;
+    let query = couchbase.analyticsQuery('SELECT * FROM default LIMIT $limit;', {"limit": limit});
+    for (let row of query) {
+    ++count;
```

Comment:
This line is not formatted correctly. It must be indented one more level because it is inside the for-loop.

You can use the following website to auto-format the JavaScript code: https://beautifier.io/